### PR TITLE
Add Spark Miner RSR buckets over time

### PR DESCRIPTION
**Pull request by juliangruber**
Screenshot 2024-11-06 at 12 02 59

Closes https://github.com/space-meridian/roadmap/issues/153

### CODE REVIEW

File: src/data/spark-miner-rsr-summaries.json.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const summaries = {};
+
+for (let i = 201; i >= 1; i--) {
+  const dayString = getDateXDaysAgo(i);
+  summaries[dayString] = await jsonFetcher(
+    `https://stats.filspark.com/miners/retrieval-success-rate/summary?from=${dayString}&to=${dayString}`
+  );
+}
+
+process.stdout.write(JSON.stringify(summaries));
```
Comment (bajtos) [inline src/data/spark-miner-rsr-summaries.json.js:13]: Forgotten log?

**Comment by bajtos** [comment]
👌🏻

File: src/data/spark-miner-rsr-summaries.json.js (hunk 1 above)
Comment (juliangruber) [reply src/data/spark-miner-rsr-summaries.json.js:13]: Nope, without this this data source wouldn't produce any data

Comment (juliangruber) [reply src/data/spark-miner-rsr-summaries.json.js:13]: These scripts are expected to write their data to stdout

Comment (bajtos) [reply src/data/spark-miner-rsr-summaries.json.js:13]: Oh, interesting. OK!

**Pull request merged**